doc/source/ray-use-cases/overview.rst

```diff
@@ -0,0 +1,61 @@
+.. _ref-use-cases:
+
+Ray Use Cases
+=============
+
+This is the start of collecting some Ray use cases. It is not a replacement for blog posts
+but is meant to be an engineering resource that you can use to build your own use cases.
+
+.. panels::
+    :container: container pb-4
+    :column: col-md-4 px-2 py-2
+    :img-top-cls: pt-5 w-75 d-block mx-auto
+
+    ---
+    :img-top: /images/taxi.png
+
+    +++
+    .. link-button:: /data/examples/nyc_taxi_basic_processing
```

Comment:
btw, I think we have a repo for docs resources to keep this one smaller. I don't think it matters _that_ much, but since we'll likely add many more resources, maybe good to lead by example.

Reply:
I recompressed the images to make them smaller -- I think for now we should optimize for iteration speed here and make it very easy to replace the images / add new ones. We can solve the scaling problem when we have it, e.g. I will be super happy when we have enough examples so we cannot afford a few hundred kilobytes of images per example in the repo any more :)

Even 100 examples with 200kb image per example will only add up to 20MB.

Fully agreed that for larger amounts of data we should upload it to S3.